nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -506,32 +506,32 @@ WaypointFollower::convertGPSPosesToMapPoses(
     request->ll_point.longitude = curr_geopose.position.longitude;
     request->ll_point.altitude = curr_geopose.position.altitude;
 
-    from_ll_to_map_client_->wait_for_service((std::chrono::seconds(1)));
```

Comment:
If not, we should delete the impl of this method and immediately return an empty vector so that it hits the error case that there are no valid waypoints.